packages/taxdat/R/report_plots.R

```diff
@@ -163,7 +154,7 @@ plot_gam_fit_input_cases <- function(config, cache, cholera_directory) {
 #' @return ggplot object
 plot_gam_fit_input_rates <- function(config, cache, cholera_directory) {
   get_initial_values_df(config = config, cache = cache, cholera_directory = cholera_directory)
-  return(plot_sf_with_fill(cache, "initial_values_df", color_scale_type = "rates", fill_column = "suspected_cases/population", geometry_column = "geometry", facet_column = "(t+1999)", legend_title = "\n Incidence rate\n"))
```

Comment:
lubridate::ymd('1999-01-01') + lubridate::period(t, config[["general"]][["time_scale"]])

for plots of simulation framework stuff you may want to use the config[["test_metadata"]][["time_scale"]]